common/ta_errors.h

```diff
@@ -124,6 +124,10 @@ typedef enum {
   /**< Failed in cache operations */
   SC_CACHE_OFF = 0x03 | SC_MODULE_CACHE | SC_SEVERITY_MINOR,
   /**< Cache server is not turned on */
+  SC_CACHE_LOCK_INIT_DESTROY = 0x04 | SC_MODULE_CACHE | SC_SEVERITY_FATAL,
+  /**< Failed to initialize or destroy lock in cache */
```

Comment:
`SC_CACHE_LOCK_INIT_DESTROY` is misleading. Can you clarify and shorten it?

Reply:
`SC_CACHE_SET_LOCK`?

Reply:
How about `SC_CACHE_LOCK_ERROR`?

Reply:
OK